rest-api/src/main/java/com/cloudbees/workflow/flownode/FlowNodeUtil.java

```diff
@@ -44,33 +61,88 @@
  */
 public class FlowNodeUtil {
 
+    protected static final Cache<String,List<FlowNode>> executionCache = CacheBuilder.newBuilder().maximumSize(100).build();
```

Comment:
:ant: `protected static` makes no sense on an uninstantiable (so, effectively `final`) class. You can just delete `protected` and it will still be accessible from within the package.